common/djangoapps/course_modes/admin.py

```diff
@@ -23,7 +23,7 @@
 from util.date_utils import get_time_display
 from xmodule.modulestore.django import modulestore
 
-COURSE_MODE_SLUG_CHOICES = [(mode_slug, mode_slug) for mode_slug in settings.COURSE_ENROLLMENT_MODES]
+COURSE_MODE_SLUG_CHOICES = [(key, enrollment_mode['display_name']) for key, enrollment_mode in settings.COURSE_ENROLLMENT_MODES.iteritems()]
```

Comment:
Please use six.iteritems() here for future Python 3 compatibility.

Reply:
@doctoryes Done in commit 10f6b8d5ac92a46032b695c5baa312925dac321f